### ISSUE TITLE
Suppress x-enumNames in swagger json

### ISSUE THREAD

**Issue by aaronhudon** [RSuter/NSwag#1748]
Is there an option to suppress the x-enumNames content from the swagger model?  This content causes AWS API Gateway to have a fit.

**Comment by RSuter** [comment]
You can implement a custom ISchemaProcessor and remove these in each generated schema

**Comment by aaronhudon** [comment]
Thanks, will do.

**Comment by aaronhudon** [comment]
> You can implement a custom ISchemaProcessor and remove these in each generated schema

This works for runtime purposes, do you have any suggestions when using Nswag.Msbuild ?  This is how my documentation is generated.

**Comment by aaronhudon** [comment]
> In v12, this is the recommended way for CLI: Create nswag.json with NSwagStudio (see screenshot) and run via CLI, https://user-images.githubusercontent.com/2603405/47795762-5d143780-dd23-11e8-849a-a56d4b1dbd42.png

Does this mean NSwag.MSBuild is obsolete?  I have opted for code-first swagger.json generation.

**Comment by RSuter** [comment]
Nope, just run the config file with NSwag.MSBuild instead of specifying everything inline

**Comment by aaronhudon** [comment]
> Nope, just run the config file with NSwag.MSBuild instead of specifying everything inline

You mean this config file, right? https://github.com/RSuter/NSwag/wiki/NSwag-Configuration-Document

**Comment by RSuter** [comment]
Yes exactly this... but the sample in the wiki might be outdated, create one with studio...

This screenshot shows the most recent recommended flow (middleware + config): 
https://github.com/RSuter/NSwag/pull/1655

**Comment by aaronhudon** [comment]
> Yes exactly this... but the sample in the wiki might be outdated, create one with studio...
> 
> This screenshot shows the most recent recommended flow (middleware + config):
> #1655

I see the command line supports `DocumentProcessors` and `OperationProcessors`, but not `SchemaProcessors` (`ISchemaProcessor`) ... is the workaround to use a `DocumentProcessor`?

**Comment by RSuter** [comment]
I think its just missing, but with document provider the cli just uses the config in AddSwaggerDocument()
